src/olympia/lib/settings_base.py

```diff
@@ -1656,6 +1656,9 @@ def get_redis_settings(uri):
 
     # Use json by default when using APIClient.
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
+
+    # Use http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.15
```

Comment:
Add a comment explaining that we can't use the default mechanism because we don't use django timezone support